src/zrc/api/tests/test_audittrails.py

```diff
@@ -44,7 +44,7 @@ class AuditTrailTests(JWTAuthMixin, APITestCase):
         }
     }
 
-    def _create_zaak(self):
+    def _create_zaak(self, **HEADERS):
```

Comment:
bit weird to see this uppercased - typically you use uppercase variable names for constants only. I'll merge it as is now, but next time please lowercase your variable name :smile: